src/renderer/src/test/helpers/ipc.ts

```diff
@@ -0,0 +1,59 @@
+import { createRequire } from 'node:module'
+import path from 'node:path'
+import { MessageChannel } from 'node:worker_threads'
+import { MapeoManager } from '@comapeo/core'
+import {
+	closeMapeoClient,
+	createMapeoClient,
+} from '@comapeo/ipc/dist/client.js'
+import { createMapeoServer } from '@comapeo/ipc/dist/server.js'
+import { KeyManager } from '@mapeo/crypto'
+import Fastify from 'fastify'
+// @ts-expect-error - no types
```

Comment:
if you add the following to the `compilerOptions` in the project root `tsconfig.json`, it should fix this issue:

```
"baseUrl": "./",
"paths": {
    "*": ["node_modules/@digidem/types/vendor/*/index.d.ts"]
}
```